### PR TITLE
Added download as app instructions to about page

### LINKED ISSUE #636: Add guide to download website as app
This could perhaps go on the About page? Most users had no idea how to download the site as an app, or they said "go to the app store"

### PR DESCRIPTION
**Pull request by akampen**
Closes #636 

Not much Spanish here, would you mind giving it a look over? If you feel confident enough I don't think we need to run it by Hillary.

### CODE REVIEW

File: public/locales/es/about.json [hunk 1]
```diff
@@ -33,6 +33,23 @@
         "part2": "Los sensores PurpleAir nos caen bien porque son económicos y relativamente precisos considerando su precio. Dicho esto, no son tan confiables como algunos sensores más caros, lo que significa que a veces los valores que producen los sensores son insuficientes para calcular el AQI. Esto podría ser muy temporal (una cuestión de minutos u horas) o podría durar más tiempo. Si un sensor ha estado inactivo durante mucho tiempo, probablemente significa que el sensor ya no está conectado al Internet o que tiene un problema de hardware más grave.",
         "part3": "Si un sensor en tu área ha estado inactivo durante mucho tiempo, por favor contáctanos y nos comunicaremos con el anfitrión del sensor para tratar de resolver el problema."
     },
+    "app":{
+        "heading": "Descargue este sitio web como una aplicación",
+        "body": "Si tiene un smartphone Android o un iPhone, puede descargar este sitio web como una aplicación en su pantalla de inicio siguiendo estas instrucciones.",
```
Comment (CarsonHerness) [inline public/locales/es/about.json:38]: Maybe use instalar instead of descargar

**Review by CarsonHerness — APPROVED** [review]

File: public/locales/es/about.json (hunk 1 above)
Comment (CarsonHerness) [reply public/locales/es/about.json:38]: Also definitely have Hillary check these, I tried checking on my phone to see what it said but now my phone language is all confused lol

**Comment by akampen** [comment]
Waiting on Hillary's review, will merge on Wednesday 4/21

**Comment by akampen** [comment]
Carson, I simply cannot figure out how to get the icon to appear on the same line as the instructions on small screens. Can you take a look?

**Comment by CarsonHerness** [comment]
> Carson, I simply cannot figure out how to get the icon to appear on the same line as the instructions on small screens. Can you take a look?

I have no idea, I suggest making an HStack (maying with `spacing={1}`) without the parenthesis around the share icon, and then we just live with the weird formatting

**Pull request merged**